docs/pages/en-us/Client-API.md

```diff
@@ -5,19 +5,24 @@ title: Client-side API
 
 # Audit Client
 
-The `caf-audit` library offers a convenient set of classes for creating a connection and sending audit events to persistent storage.
+The `caf-audit` library offers a convenient set of classes for creating a connection and sending audit events to an endpoint such as a Web Service or storage.
+
+These are the following `caf-audit` client-side API connection modes provided by the library:
+
+- [Direct to Elasticsearch](#direct-to-elasticsearch-configuration) mode connects directly to Elasticsearch for the storage of audit event messages.
+- [Web Service Client](#audit-web-service-client-configuration) mode connects to the Audit Web Service REST API for the messaging of audit events. The Audit Web Service stores audit event messages into Elasticsearch.
+- [No-op](#no-op) mode which does not connect to an endpoint or build audit event messages. This mode can be useful for testing as Elasticsearch or Audit Web Service components are not required.
 
 ## Using Auditing Objects
 
 This section covers `caf-audit` library classes and how to use them as objects within your application.
 
 The order of instantiation and use of these objects for sending audit events is as follows:
 
-1. If you do not have an existing [`ConfigurationSource`](#ConfigurationSource) object, create one for retrieving and holding configuration details.
-2. Create an [`AuditConnection`](#AuditConnection) object by passing the `AuditConnectionFactory` the [`ConfigurationSource`](#ConfigurationSource).
-3. Use the `AuditConnectionFactory` to create the [`AuditConnection`](#AuditConnection) object. You will need to pass it the [`ConfigurationSource`](#ConfigurationSource).
+1. If you do not have an existing [`ConfigurationSource`](#ConfigurationSource) object, create one for retrieving and holding configuration details for the mode that you require.
+2. Use the [`AuditConnectionFactory`](#AuditConnectionFactory) to create the [`AuditConnection`](#AuditConnection) object by setting the [`CAF_AUDIT_MODE Environment Variable`](#CAF_AUDIT_MODE-environment-variable) and pass it the [`ConfigurationSource`](#ConfigurationSource).
 3. Create an [`AuditChannel`](#AuditChannel) object from the [`AuditConnection`](#AuditConnection) object.
-4. Use the [`AuditEventBuilder`](#AuditEventBuilder) object to construct audit events and send them to Elasticsearch.
+4. Use the [`AuditEventBuilder`](#AuditEventBuilder) object to construct and send audit event messages to the endpoint.
 
 ### ConfigurationSource
```

Comment:
As discussed the ConfigurationSource section covers how to read in configuration from file.